backend/src/prompts/prompt_templates.py

```diff
@@ -0,0 +1,74 @@
+summarise_prompt_template = """
+You are an expert programmer and problem-solver, tasked with answering any question about the OpenROAD (OR) project and the OpenROAD-Flow-Scripts (ORFS).
```

Comment:
why did you use backslash for triple quoted strings for above files and not this file?

Reply:
That was purely to improve the readability of the longer prompts

Reply:
This is the purpose of triple quoted strings to lineate your long strings neatly. You could separate strings if they are getting too long